musicrs/util/youtubeIdToLink.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+def youtubeIdToLink(basePath):
```

Comment:
python follows snake case for defining function name.. take reference from other function names in repo

Reply:
:ok: